website/scripts/publish.sh

```diff
@@ -2,6 +2,4 @@
 
 cp static/package.json build/
 cd build
-echo $(ls ../../kernel/scripts)
-npx tsc --build ../../kernel/scripts/tsconfig.json
-node ../../kernel/scripts/npmPublish.js
+npx -T ../scripts/npmPublish.ts
```

Comment:
What does `npx -T` do?

Reply:
It does nothing on it's own, it was a typo, already sent the fix on another commit

Reply:
It was `npx ts-node -T ...` so that ts-node doesn't validate typescript errors and just transpiles the code